nix/modules/devshell.nix

```diff
@@ -38,6 +38,7 @@ in
         ''
           # For nixci
           export DEVOUR_FLAKE=${inputs.devour-flake}
+          export OMNIX=${inputs.omnix}
```

Comment:
This should be in around line 20 above.

I'll move DEVOUR_FLAKE there as well later.